src/utils.js

```diff
@@ -210,11 +205,45 @@ go.utils = {
         });
     },
 
-    subscription_update_language: function(contact, im) {
-        // TODO patch subscription language
-        return Q();
+    subscription_update_language: function(im, contact) {
+        var params = {
+            to_addr: contact.msisdn
+        };
+        return go.utils
+        .control_api_call("get", params, null, 'subscription/', im)
```

Comment:
The underlying call now needs updating to be using Token auth not the current old TastyPie way